src/crawlers/cheerio_crawler.js

```diff
@@ -552,6 +597,30 @@ class CheerioCrawler {
             }
         });
     }
+
+    /**
+     * Handles blocked request
+     * @param session {Session}
+     * @param statusCode {Number}
+     * @private
+     */
+    _handleBlockedRequest(session, statusCode) {
```

Comment:
I think `_throwOnBlockedRequest` would be more descriptive. `_handleBlockedRequest` suggests that we already know the request is blocked when calling it.